examples/sqlrooms-ai/src/store.ts

```diff
@@ -120,6 +121,14 @@ export const {projectStore, useProjectStore} = createProjectStore<
             structuredOutputs: true,
           });
         },
+        getApiKey: () => {
+          return get()?.openAiApiKey || '';
+        },
+        getTableSchema: async () => {
```

Comment:
getTableSchemas?

Could this be part of a `project` or `duckdb` slice?

Reply:
There's already `getDuckTableSchema` in `@sqlrooms/duckdb`:
https://github.com/sqlrooms/sqlrooms/blob/main/packages/duckdb/src/useDuckDb.ts#L202 
Maybe we can reuse it